app/models.py

```diff
@@ -260,6 +260,8 @@ class SupplierFramework(db.Model):
                              db.ForeignKey('frameworks.id'),
                              primary_key=True)
     declaration = db.Column(JSON)
+    on_framework = db.Column(db.Boolean, nullable=True)
```

Comment:
This isn't very clear, maybe it should be documented in the commit message for https://github.com/alphagov/digitalmarketplace-api/commit/c2227c7512c130c838f18a1a87a19bad45ea85ba